rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,175 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Deploying the OSToy application with Kubernetes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+Deploying an application involves creating a container image, storing it in an image repository, and defining Deployment object that uses that image.
+
+Deploying an application involves the following steps:
+* Create the images for the front-end and back-end microservice containers
```

Comment:
You need to insert a new line at line 15. It doesn't render correctly currently:

![image](https://github.com/user-attachments/assets/7c07fc88-f895-4ee7-86ee-fc3febb6046c)